src/darwin/mod.rs

```diff
@@ -572,4 +516,9 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn get_font_smoothing_enabled() {
+        println!("font smoothing enabled: {}", *super::FONT_SMOOTHING_ENABLED);
```

Comment:
It was very helpful during development to make sure I was using the `objc` stuff correctly, rather than a separate binary that just read this value. If you feel strongly I can remove it.

Reply:
I'd say it tests that the font smoothing thunk does not panic or crash, which is a possibility given that it's a medium-sized `unsafe` block. I'll figure out a way to turn this into a proper test though, to make sure different cases of user defaults are handled correctly.

Reply:
Never mind. This thunk is executed already in the existing test, which calls `get_glyph`. There's not much value beyond that, so I will remove.